API/WithDocs/Microsoft.Extensions.AsyncState/IAsyncLocalContext.cs

```diff
@@ -0,0 +1,17 @@
+// Assembly 'Microsoft.Extensions.AsyncState'
+
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Extensions.AsyncState;
+
+/// <summary>
+/// Provides access to the current async context stored outside of the HTTP pipeline.
```

Comment:
Should we really mention `HTTP` here? The package is agnostic

Reply:
Good point, I'll remove.